board/grinn/liteboard-relay/imx6ul-liteboard-relay.dts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016 Grinn
+ *
+ * Author: Piotr L. Figlarek <piotr.figlarek@grinn-global.com>
```

Comment:
I'm not an author of this file. Please correct it and push new PR.

Reply:
Sorry for that mistake. New PR has been prepared.